av/src/Video.cc

```diff
@@ -289,3 +289,10 @@ int Video::Height() const
 {
   return this->dataPtr->codecCtx->height;
 }
+
+/////////////////////////////////////////////////
+std::chrono::steady_clock::duration Video::Duration() const
+{
+  return std::chrono::duration<int64_t, std::ratio<1, AV_TIME_BASE>>(
+      this->dataPtr->formatCtx->duration);
```

Comment:
I think it would make sense to typedef this definition and return that type here.

The duration that we are returning isn't a `steady_clock` duration here.

Reply:
Would this be ok? https://github.com/ignitionrobotics/ign-common/blob/10b186592ccf650af373ba8f4a485713081ab530/av/include/ignition/common/Video.hh#L61-L68

Reply:
I don't mind to switch it to public but isn't just the alias private and what is being returned is `std::chrono::duration`?

Reply:
I believe it's the full type that's being returned, but as an example in this case:

```
// Works
auto dt = v.Duration();

// Works
std::chrono::duration<int64_t, std::ratio<1, 1000000>> dt = v.Duration();

// Works
auto millis = std::chrono::duration_cast<std::chrono::milliseconds>(v.Duration());

// Doesn't work
Video::Length dt = v.Duration();

// Doesn't work
std::chrono::milliseconds dt = v.Duration();
```

I think that the 4th case is somewhat convenient in the case that you may want to use auto?

Reply:
Also, people who use either `auto` or `Video::Length` shouldn't get burned if the definition of `Length` changes.

Reply:
Ah I see, thanks. I made the changes here: c6f76c8a7f6e1c0355538b95f29c08feb1f24d00